src/Main.java

```diff
@@ -265,10 +264,8 @@ private static String ReadInput(){
             System.out.print(")\n");
             input = Scan.nextLine().toUpperCase();
             for(int i = 0; i < ValidInput.size(); i++){
-
```

Comment:
I believe our coding standards have a line in between any curly braces so I think this line should be here